src/Uploader/test/UploadFileItemSpec.tsx

```diff
@@ -14,32 +14,31 @@ const file = {
 } as const;
 
 describe('UploadFileItem', () => {
+  testStandardProps(<UploadFileItem file={file} />);
   it('Should output a UploadFileItem', () => {
-    const instance = getDOMNode(<UploadFileItem file={file} />);
-    assert.include(instance.className, 'rs-uploader-file-item');
+    const { container } = render(<UploadFileItem file={file} />);
+    expect(container.firstChild).to.have.class('rs-uploader-file-item');
   });
 
   it('Should render picture-text for layout', () => {
-    const instance = getDOMNode(<UploadFileItem listType="picture-text" file={file} />);
-    assert.include(instance.className, 'rs-uploader-file-item-picture-text');
```

Comment:
This assertion is still missing. Same for the first assertion in the next test case.